debian/changelog

```diff
@@ -1,3 +1,23 @@
+cloud-init (24.3.1-0ubuntu0~24.04.1) noble; urgency=medium
+
+  * d/p/no-single-process.patch: Remove single process optimization
+  * d/p/no-nocloud-network.patch: Remove nocloud network feature
+  * Upstream snapshot based on upstream/main at acf04d61.
+  * refresh patches:
+    - d/p/no-nocloud-network.patch
+  * Upstream snapshot based on 24.3.1. (LP: #2079224).
+    List of changes from upstream can be found at
+    https://raw.githubusercontent.com/canonical/cloud-init/24.3.1/ChangeLog
+
+ -- Chad Smith <chad.smith@canonical.com>  Thu, 05 Sep 2024 12:30:45 -0600
+
+cloud-init (24.2-0ubuntu1~24.04.2) noble; urgency=medium
+
+  * Declare breaks on the python3-minimal version that is affected by the
+    py3clean failure when using alternate character set (LP: #2075337)
+
+ -- Benjamin Drung <bdrung@ubuntu.com>  Wed, 07 Aug 2024 20:54:01 +0200
+
```

Comment:
Note, this is sync'd from direct release of 24.2-0ubuntu1~24.04.2 to the distribution packaging branches for ubuntu/noble-proposed/updates